pkg/cmd/generate.go

```diff
@@ -28,7 +28,12 @@ import (
 	"k8s.io/client-go/pkg/runtime"
 )
 
-func Generate(files []string) error {
+func Generate(paths []string) error {
+
+	files, err := GetAllYAMLFiles(paths)
```

Comment:
it was done like this in the first place, but then I changed that, to this.
It's closer to application logic and I think that in cmd/* should be mostly the code that handles argument parsing and validation.